aptos-move/aptos-gas/src/transaction/mod.rs

```diff
@@ -138,6 +165,59 @@ impl TransactionGasParameters {
         }
     }
 
+    /// New formula to charge storage fee for write set items based on fixed APT costs.
+    pub fn calculate_write_set_storage_fee<'a>(
+        &self,
+        ops: impl IntoIterator<Item = (&'a StateKey, &'a WriteOp)>,
+    ) -> Fee {
+        let excess_fee = |size: NumBytes| -> Fee {
+            match size.checked_sub(self.free_write_bytes_quota) {
+                Some(excess) => excess * self.storage_fee_per_excess_state_byte,
+                None => 0.into(),
+            }
+        };
+
+        let mut fee = Fee::zero();
+        let mut new_slots = NumSlots::zero();
+
+        for (key, op) in ops {
+            if let WriteOp::Creation(..) = op {}
```

Comment:
Why do we have an empty if here?

Reply:
Good catch. Forgot to get this deleted. It's now fixed with an update.